src/FortTroopCalculator.tsx

```diff
@@ -0,0 +1,394 @@
+                    className="mt-1 block w-full rounded-md border border-gray-300 bg-white py-2 px-3 shadow-sm focus:border-sky-500 focus:outline-none focus:ring-sky-500 sm:text-sm"
+                  >
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                  </select>
+                </div>
+                <div className="col-span-6 sm:col-span-4">
+                  <label
+                    htmlFor="totalTroops"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    Total Troops
+                  </label>
+                  <input
+                    onChange={(e) =>
+                      setTroops({
+                        ...troops,
+                        totalTroops: parseFloat(e.target.value),
+                      })
+                    }
+                    type="number"
+                    step="25000"
+                    min="0"
+                    max="375000"
+                    name="totalTroops"
+                    id="totalTroops"
+                    placeholder="200000"
+                    value={troops.totalTroops}
+                    className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-sky-500 focus:ring-sky-500 sm:text-sm"
+                  />
+                </div>
+                <div className="col-span-6 sm:col-span-4">
+                  <label
+                    htmlFor="t5Percent"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    T5 Percent
+                  </label>
+                  <input
+                    onChange={(e) =>
+                      setTroops({
+                        ...troops,
+                        t5Troops: parseFloat(e.target.value),
+                      })
+                    }
+                    type="number"
+                    step="10"
+                    min="0"
+                    value={troops.t5Troops}
+                    max="100"
+                    placeholder="60"
+                    name="t5Percent"
+                    id="t5Percent"
+                    className="mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-sky-500 focus:ring-sky-500 sm:text-sm"
+                  />
+                </div>
+              </div>
+              <div className="bg-gray-50 px-4 py-3 text-right sm:px-6">
+                {result.t5inf === 0 && (
```

Comment:
isnt this wrong? it should be the same as the check for the button?

Reply:
It was a quick fix to make it work, will change it as soon as i can get to it!

Reply:
Sounds good! i ll wait for it!